repository/RediStick-Core.package/RsRedisEndpoint.class/instance/sScanKey.cursor..st

```diff
@@ -0,0 +1,3 @@
+commands-sets
+sScanKey: aKey cursor: aCursor 
```

Comment:
For consistency to other command-set methods, 'Key:' part is not needed. And we should also support optional parameters.
```
sScan: aKey curstor: index count: count match: matchPattern
sScan: aKey curstor: index match: matchPattern
sScan: aKey curstor: index count: count
sScan: aKey curstor: index
```